src/app/pages/home/profile/profile.page.ts

```diff
@@ -66,7 +65,6 @@ export class ProfilePage implements OnInit {
           this.card.frontImg = this.router.getCurrentNavigation().extras.state.data;
       };
 
-      this.users = this.userService.getUsers();
       this.cards = this.cardService.getCards();
```

Comment:
Will have to add a method to get cards belonging to a specific user (current user's uid)

Reply:
Yes exactly!

Reply:
Yes exactly!